common/events/events_io.cpp

```diff
@@ -131,9 +131,10 @@ mio::ExpectedSize ReaderFromAsyncReader::Read(
 	vector<uint8_t>::iterator start, vector<uint8_t>::iterator end) {
 	size_t read;
 	error::Error err;
-	err = reader_->AsyncRead(start, end, [this, &read, &err](size_t n, error::Error inner_err) {
+	error::Error inner_err;
+	err = reader_->AsyncRead(start, end, [this, &read, &inner_err](size_t n, error::Error err) {
```

Comment:
I'd squash this with the commit that introduces the confusing version.

Reply:
Will do!